src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -86,11 +86,19 @@ export class OperationDetails {
                 operationPath += operation.displayUrlTemplate;
             }
 
+            let requestUrl = "";
+
             if (api.type === TypeOfApi.webSocket) {
```

Comment:
Not sure why we omit protocol for websockets here, there should be `ws` or `wss`. @ygrik can you please comment?

Reply:
We show URL and protocols separately.
![image](https://user-images.githubusercontent.com/8660244/141345932-5322dfab-b2aa-40a3-b55d-c113a515c25e.png)